docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -646,6 +646,108 @@
     "tab.mode(dropna=False)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b248fef1",
+   "metadata": {},
+   "source": [
+    "### Table.sem()\n",
+    "\n",
+    "```\n",
+    "Table.sem(axis=0, skipna=True, numeric_only=False, ddof=0)\n",
+    "```\n",
+    "Return unbiased standard error of the mean over requested axis. Normalized by N-1 by default. This can be changed using the ddof argument\n",
+    "\n",
+    "**Parameters:**\n",
+    "\n",
+    "| Name         | Type | Description                                                                      | Default |\n",
+    "| :----------: | :--: | :------------------------------------------------------------------------------- | :-----: |\n",
+    "| axis         | int  | The axis to calculate the sum across 0 is columns, 1 is rows.                    | 0       |\n",
+    "| skipna       | bool | not yet implemented                                           | True    |\n",
+    "| numeric_only | bool | Only use columns of the table that are of a numeric data type.                   | False   |\n",
+    "| ddof    | int  | Delta Degrees of Freedom. The divisor used in calculations is N - ddof, where N represents the number of elements. | 1 |\n",
+    "\n",
+    "**Returns:**\n",
+    "\n",
+    "| Type               | Description                                                          |\n",
+    "| :----------------: | :------------------------------------------------------------------- |\n",
+    "| Table         | The std across each row / column with the key corresponding to the row number or column name. |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71bd1d6f",
+   "metadata": {},
+   "source": [
+    "**Examples**\n",
+    "\n",
+    "Calculate the sem across the columns of a table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "350c2b7c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab = kx.Table(data=\n",
+    "        {\n",
+    "            'a': [1, 2, 2, 4],\n",
+    "            'b': [1, 2, 6, 7],\n",
+    "            'c': [7, 8, 9, 10],\n",
+    "            'd': [7, 11, 14, 14],\n",
+    "        }\n",
+    "    )\n",
+    "tab"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b89307e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab.sem()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6933f01f",
+   "metadata": {},
+   "source": [
+    "Calculate the sem across the rows of a table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3edd3feb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab.sem(axis=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae7afe5a",
+   "metadata": {},
+   "source": [
+    "Calculate std accross columns with ddof=0:"
```

Comment:
We are not calculating the std here, right?

Reply:
👍